woocommerce/woocommerce-android
# [Themes] Theme picker skip button

Fixes #10344. The PR fixes the theme picker Skip button functionality & hides it when the theme picker is opened from the settings.

**To test:**

Test 1
1. Start the store creation and navigate to the the picker
2. Notice the Skip button is visibe
3. Tap on the Skip button an verify the installation proceeds

Test 2
1. Go to Settings -> Tap on Themes
2. Notice the Skip button is not visible
